docs/src/examples/components/Toolbar/Usage/ToolbarExampleWithTooltip.shorthand.tsx

```diff
@@ -0,0 +1,282 @@
+import * as React from 'react'
```

Comment:
This example is super huge. How about to make it with few items (2-3) just to showcase the usage of a tooltip. Because as a client I can get lost to find the logic where the tooltip is rendered

Reply:
I simplified this a bit, but still I want to have example for each tooltip item: toggle, divider, item with popup, with menu, radio group etc. These are the options we have now:
![image](https://user-images.githubusercontent.com/4512430/61222328-c9e92980-a71a-11e9-8810-b40c0aac2b01.png)

Let me know if you want some of there to be removed, and will update it

Reply:
I agree with @sophieH29 maybe this should be a prototype... 
@mnajdova , maybe we can shorten the example and make it point to the prototype (with a link) for more details since we plan to have a public prototypes section anyway.

Reply:
agree - in its current form this example is rather a prototype. 

Would suggest to avoid `group` complications here, just place three or four items in the `Toolbar` and add tooltip to them - this should perfectly make a `Toolbar example with tooltip` :)

> but still I want to have example for each tooltip item: toggle, divider, item with popup, with menu, radio group etc

As it is the same logic used for any kind of tooltip item, I think it would be enough to provide example for one kind of item only, and just say in the comments that all other kind of items could be treated the same way.

Reply:
Thanks for the suggestions guys, I refactored the example, it's much shorter now, please take a look!